plugins/trezor.py

```diff
@@ -230,10 +230,22 @@ def create_main_account(self, password):
 
     def mnemonic_to_seed(self, mnemonic, passphrase):
         # trezor uses bip39
-        import pbkdf2, hashlib, hmac
+        import hashlib, hmac
         PBKDF2_ROUNDS = 2048
         mnemonic = ' '.join(mnemonic.split())
-        return pbkdf2.PBKDF2(mnemonic, 'mnemonic' + passphrase, iterations = PBKDF2_ROUNDS, macmodule = hmac, digestmodule = hashlib.sha512).read(64)
+
+        try:
+            from Crypto.Protocol import PBKDF2
```

Comment:
Blame on me, it should be. But I don't have a Trezor to test it, so It got unnoticed by me.